### PR TITLE
chore: update github action to build and push containers

### PR DESCRIPTION
**Pull request by LucaLanziani**

### CODE REVIEW

File: assets/github/pipeline.tmpl [hunk 1]
```diff
@@ -9,16 +9,22 @@ on:
...
     runs-on: ubuntu-latest
-    container: 
-      image: ghcr.io/nearform/k8s-kurated-addons-cli:latest
     steps:
+      - uses: actions/checkout@v2
+
       - name: build docker image application
-        run: build
-      
+        uses: docker://ghcr.io/nearform/k8s-kurated-addons-cli:latest
```
Comment (zimny) [inline assets/github/pipeline.tmpl:16]: is there some benefit of using docker:// vs container.image property?

Comment (LucaLanziani) [reply assets/github/pipeline.tmpl:16]: Container Image doesn't work 😅

Comment (LucaLanziani) [reply assets/github/pipeline.tmpl:16]: I just found another solution that might work, I'll try it later

**Review by zimny — APPROVED** [review]
ok, lgtm

**Pull request merged**